src/web3_extentions/lido_validators.py

```diff
@@ -63,17 +74,22 @@ def get_lido_validators_by_node_operators(self, blockstamp: BlockStamp) -> Dict[
 
     @lru_cache(maxsize=1)
     def get_lido_node_operators(self, blockstamp: BlockStamp) -> list[OperatorExpanded]:
-        operators_by_modules: list[OperatorResponse] = self.w3.kac.get_operators(blockstamp)
 
         operators = []
 
-        for module in operators_by_modules:
-            operators.extend([
-                OperatorExpanded(
-                    stakingModuleAddress=module.module.stakingModuleAddress,
-                    **asdict(operator),
-                )
-                for operator in module.operators
-            ])
+        staking_modules = self.w3.lido_contracts.staking_router.functions.getStakingModules().call(
```

Comment:
Let's move contract call to separate method and add there comment with response type